app/models/user.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      firstName: {
+        allowNull: false,
+        field: 'first_name',
+        type: DataTypes.STRING
+      },
+      lastName: {
+        allowNull: false,
+        field: 'last_name',
+        type: DataTypes.STRING
+      },
+      email: {
+        allowNull: false,
+        type: DataTypes.STRING,
+        unique: true,
+        validate: {
+          isEmail: true,
+          contains: '@wolox.com.ar'
```

Comment:
what happend if my email is: wolox.co/cl/com?

Reply:
haha it's true, now I change it